src/lib/ts/tsconfig.ts

```diff
@@ -38,7 +38,7 @@ export const initializeTsConfig = (defaultTsConfig: TsConfig, entryPoint: NgEntr
   tsConfig.options.basePath = basePath;
   tsConfig.options.baseUrl = basePath;
   tsConfig.options.rootDir = basePath;
-  tsConfig.options.outDir = outDir;
+  tsConfig.options.outDir = basePath;
```

Comment:
Seting `outDir`equal to `basePath` just results in a project-relative path for the triple-slash reference in the result d.ts